test/test_model.py

```diff
@@ -1484,87 +1555,202 @@ def test_unit_get_container(self):
             unit.get_container('c3')
 
         with pytest.raises(RuntimeError):
-            other_unit = self.model.get_unit('other')
+            other_unit = model.get_unit('other')
             other_unit.get_container('foo')
 
 
-class TestContainerPebble(unittest.TestCase):
-    def setUp(self):
+class MockPebbleBackend(_ModelBackend):
```

Comment:
These two classes are moved above where they are used.

Reply:
Not a bad idea to move it, but can we avoid moving it in this PR (increasing the diff size of an already-large diff)?